file.go

```diff
@@ -23,11 +23,24 @@ func MarshalD3ToMemory(d3Data D3) {
 	D3GraphData = strings.Replace(string(d3JSON), "null", "{}", -1)
 }
 
+// GetLastUpdateTime fetches the last time graph data (see: `D3` structure) was updated into a RFC1123Z formatted string
+func GetLastUpdateTime() string {
+	var d3Data D3
+	d3RawGraphData := strings.Replace(D3GraphData, "{}", "null", -1)
```

Comment:
But it seems it is not valid for Go JSON unmarshalling (I haven't checked this thorough, just saw it didn't work, saw the lines just above for marshalling and wrote the reverse function)